### DISCUSSION TITLE
update request to the latest NUT version

### DISCUSSION THREAD

**Discussion by o-l-o** [hassio-addons/app-nut#233] [General]
Would it be possible to update this add-on to the latest NUT version (2.8)?

**Comment by frenck** [comment]
The addon currently runs the latest available version offered by Debian.

**Reply by rcdailey** [reply]
I don't understand why there's a dependency on Debian. Based on their package update history, it doesn't look like we can anticipate an update anytime soon. It would be nice if NUT could be facilitated independent of what Debian has.

**Reply by frenck** [reply]
We did that in the past, but wasn't that great either.

If you need a newer version; this add-on is unfortunately not suitable for you.
